app/views/pins/show.html.erb

```diff
@@ -1,6 +1,7 @@
 <p id="notice"><%= notice %></p>
 
 <p>
+  <%= link_to image_tag(@pin.image.url, class: 'media-object'), @pin.image.url, target: '_blank' %>
```

Comment:
This is just to test that the image upload was working. You'd probably want to display a thumbnail version of the image instead what this is doing, which is showing the original.